### PR TITLE
Recipepage design

### LINKED ISSUE #5: Implement adding page design
Implement adding page design using templates according to wireframes design

### PR DESCRIPTION
**Pull request by EwaTrzemzalska**
Resolves #5

### CODE REVIEW

File: addrecipe.html [hunk 1]
```diff
@@ -17,24 +17,38 @@
...
+  </nav>
+
 
+  <!-- <div class="uk-margin">
+    <form class="uk-search uk-search-default">
+      <label for="searching" class="size-10">WHAT ARE YOU LOOKING FOR?</label>
+      <a href="" class="uk-search-icon-flip" uk-search-icon></a>
+      <input class="uk-search-input" type="search" name="searching" placeholder="Search...">
+    </form>
+  </div> -->
```
Comment (smogg) [inline addrecipe.html]: Do we need this comment in the diff?

File: addrecipe.html [hunk 2]
```diff
@@ -17,24 +17,38 @@
...
+
+  <form class="uk-search uk-search-default margin-20">
     <div class="uk-margin">
-        <form class="uk-search uk-search-default">
-            <a href="" class="uk-search-icon-flip" uk-search-icon></a>
-            <input class="uk-search-input" type="search" placeholder="Search...">
-        </form>
+      <label for="searching" class="size-13">WHAT ARE YOU LOOKING FOR?</label>
```
Comment (smogg) [inline addrecipe.html]: What does the `for` attribute do? What does it refer to?

File: addrecipe.html [hunk 3]
```diff
@@ -17,24 +17,38 @@
...
+      <a href="#" class="uk-search-icon-flip" uk-search-icon></a>
+      <input class="uk-search-input" type="search" placeholder="Search...">
     </div>
+  </form>
+
+  <div>
+    <form>
 
+    </form>
+  </div>
```
Comment (smogg) [inline addrecipe.html]: Why do we have this empty form here?

File: styles/style.styl [hunk 4]
```diff
@@ -1,9 +1,12 @@
...
-    padding: 20px;
-
 .mybutton
     margin: 10px;
 
 .uk-search-default
     width: 200px;
+
+.size-13
+    font-size: 13px;
```
Comment (smogg) [inline styles/style.styl]: Stylus provides variables. To make the project consistent, I recommend creating one namespace with variables, importing it in the main `.styl` file and then setting the font-sizes using those variables.

File: styles/style.styl [hunk 5]
```diff
@@ -1,9 +1,12 @@
...
+
+.margin-20
```
Comment (smogg) [inline styles/style.styl]: This is pretty specific name refering to the style used. Semantically this name should be a description of what the class does (which it is) but you shouldn't need to update the class name every time you decide to change something. Let's say you have 50 elements with a class of `.margin-20` in a month and you decide to make that spacing 25px instead. You'd have to change the class name to `margin-25` in 50 places. If you named the class something like `margin-medium` then you'd only have to change a single line in your code to affect 50 different components.

**Comment by smogg** [comment]
When you get a chance, read a short intro about CSS methodologies (different ways of coming up with class names for your project): https://www.hongkiat.com/blog/css-writing-methodologies/

File: addrecipe.html (hunk 1 above)
Comment (EwaTrzemzalska) [reply addrecipe.html]: No, I will delete it.

File: addrecipe.html (hunk 3 above)
Comment (EwaTrzemzalska) [reply addrecipe.html]: Mistake

File: addrecipe.html (hunk 2 above)
Comment (EwaTrzemzalska) [reply addrecipe.html]: The first element in the document with an id matching the value of the for attribute is the labeled control for this label element, if it is a labelable element. But it's not necessary here.

File: styles/style.styl (hunk 4 above)
Comment (EwaTrzemzalska) [reply styles/style.styl]: Fixed

File: styles/style.styl (hunk 5 above)
Comment (EwaTrzemzalska) [reply styles/style.styl]: You're right, fixed

File: addrecipe.html [hunk 6]
```diff
@@ -17,24 +17,25 @@
...
+  </nav>
+
 
+  <form class="uk-search uk-search-default margin-medium">
     <div class="uk-margin">
-        <form class="uk-search uk-search-default">
-            <a href="" class="uk-search-icon-flip" uk-search-icon></a>
-            <input class="uk-search-input" type="search" placeholder="Search...">
-        </form>
+      <label class="font-size-medium">WHAT ARE YOU LOOKING FOR?</label>
```
Comment (smogg) [inline addrecipe.html]: Just an FYI - the `for` part is important for accessibility reasons. It's fine if we add it in a future PR

File: addrecipe.html [hunk 7]
```diff
@@ -17,24 +17,25 @@
...
+      <a href="#" class="uk-search-icon-flip" uk-search-icon></a>
```
Comment (smogg) [inline addrecipe.html]: I'm a little surprised they use an empty link here, but it seems this is what they do on the framework's site ¯/\(°_o)/¯

File: styles/style.styl [hunk 8]
```diff
@@ -1,9 +1,13 @@
-
-form 
-    padding: 20px;
+fontSizeMedium = 12px;
 
 .mybutton
```
Comment (smogg) [inline styles/style.styl]: In future PR, change that class name - it doesn't seem to describe what the class isd oing at all. Also, consider adding some general-purpose classes for spacing you can use on any element, not only on a button (i.e.: like `.margin-top-medium` or `.mtm` etc)

File: addrecipe.html (hunk 6 above)
Comment (EwaTrzemzalska) [reply addrecipe.html]: Ok, I create an issue here https://github.com/EwaTrzemzalska/zjemto/issues/34

File: addrecipe.html (hunk 7 above)
Comment (EwaTrzemzalska) [reply addrecipe.html]: https://github.com/EwaTrzemzalska/zjemto/issues/34

File: styles/style.styl (hunk 8 above)
Comment (EwaTrzemzalska) [reply styles/style.styl]: https://github.com/EwaTrzemzalska/zjemto/issues/34

**Pull request merged**